app/plugins/logging.js

```diff
@@ -1,11 +1,11 @@
 import HapiPino from 'hapi-pino'
 
-const plugin = {
+const logging = {
```

Comment:
Name changed as SonarCloud points out that name of the default export and the file should match.